framework/doc/content/source/transfers/MultiAppCloneReporterTransfer.md

```diff
@@ -0,0 +1,52 @@
+# MultiAppCloneReporterTransfer
+
+!syntax description /Transfers/MultiAppCloneReporterTransfer
+
+## Overview
+
+This transfer is very similar to [MultiAppReporterTransfer.md] with a few key differences. The first is that this object can only transfer from sub-application(s) to the main app. The second is that reporter values are declared on-the-fly, i.e. the reporter values from the sub-app are "cloned" onto the main app, with out an already existing value on the main app. The third is that when there are multiple sub-applications, the values are collected into a vector of that reporter type on the main app.
+
+!alert warning
+It is not possible to transfer a reporter value that has already been cloned with this object. 
+
+## Example Input Syntax
+
+### Multiple Sub Applications
+
+This example shows a multi-app with multiple sub applications. It uses the same input file, but with different reporter values controlled by [!param](/MultiApps/TransientMultiApp/cli_args):
+
+!listing reporter_transfer/clone.i block=multi_reporter indent=2 header=[MultiApps] footer=[]
+
+The transfer syntax is pretty minimal giving reporter value names (can be found in [sub0.i](reporter_transfer/sub0.i)) on the sub-app and reporter object name on main app, the object name is used to associate the cloned value to an object.
+
+!listing reporter_transfer/clone.i block=multi_rep indent=2 header=[Transfers] footer=[]
+
+When run in serial, a single output file is produced which shows the vectors of the reporter values created:
+
+!listing reporter_transfer/gold/clone_serial.json language=json start=multi_rep:from_sub_pp:value end=multi_vpp:from_sub_vpp:a
+
+In parallel, the sub app reporter values are split into different files, one for each processor. This example was run with 6 processors which split the 4 sub apps. So the root processor for each sub app ([clone_out.json](reporter_transfer/gold/clone_out.json), [clone_out.json.1](reporter_transfer/gold/clone_out.json.1), [clone_out.json.2](reporter_transfer/gold/clone_out.json.2), and [clone_out.json.4](reporter_transfer/gold/clone_out.json.4)) will show the value, while the other processors ([clone_out.json.3](reporter_transfer/gold/clone_out.json.3) and [clone_out.json.5](reporter_transfer/gold/clone_out.json.5)) will be empty:
```

Comment:
Agreed, this what I was trying to describe to you on slack, but failed.

Reply:
This should be able to handled by the Context object. Perhaps the aforementioned Receiver object would have a flag that would cause the data to be collected on the root if set.

Reply:
I was thinking this could be handled by adding a consumer mode to the context. Maybe we can add a parameter in Reporter.C like `serialize_on = ROOT/REPLICATED/NONE` then in the declare methods, we add a consumer mode based on these values.

Reply:
If it is just for output, maybe we can just have a custom store method for ReporterVectorContext.

Reply:
Future problem. Thanks.